events/api.py

```diff
@@ -558,12 +558,14 @@ def validate_data_source(self, value):
     def validate_publisher(self, value):
         if value:
             # TODO: allow optionally declaring publisher in any user admin org (regular org for drafts)
-            if value not in (self.publisher, self.publisher.replaced_by):
+            # TODO: self.user.admin_organizations.all() or user.get_admin_organizations_and_descendants()?
```

Comment:
`self.user.admin_organizations.all()` should be enough, because we do not need users actually publishing new events on behalf of their descendants.

Descendants must be allowed, however, in existing events users from the top level organizations do edit. Therefore, extra check will have to be done by me so that validation at PUT will be done differently, as per issue #305 . So this is fine for now.

Reply:
Damn! We will need `user.get_admin_organizations_and_descendants()` after all, this will make implementing #305 a breeze :D